content/en/service_owners_guide/plan.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Plan your Datadog implementation
+description: Setup and implement Datadog as a Service owner to avoid pit-falls down the road
+further_reading:
+- link: "/getting_started/tagging/unified_service_tagging/"
+  tag: "Documentation"
+  text: "Getting Started with Unified Service Tagging"
+- link: "/getting_started/tracing/"
+  tag: "Documentation"
+  text: "Getting Started with APM Tracing"
+---
+
+## Design
+
+### Sizing exercise
+
+Defining a clear end goal is essential when starting a significant product implementation. However, it's impractical to know all requirements from the beginning. Product engineers iterate deployments, and systems operations manage changes to mitigate risk. Similarly, a large-scale Datadog deployment benefits from standard project management practices. Certain Datadog elements should be included in this process. Survey outlines are useful for assessing and planning your needs.
+
+A sample survey form might look like this: 
+
+*Application name:*  
+	*Language:*  
+		*Frameworks:*  
+	*Model Layer:*   
+	*View Layer:*  
+	*Controller layer:*  
+	*Infra Type:*  
+	*Operating systems:*
+
+**Recommendation:**   
+Start whiteboarding early, collecting or consolidating a mental survey of your outline. Create a comprehensive view of your ecosystems, application language, data storage, networking, and infrastructure.
+
+## General best practices 
+
+Datadog has specialized tool-tips designed for every kind of material.  After completing the sizing exercise, you will understand the types of technologies you're working with, and can start mapping those to core products in Datadog.
+
+### Resource tagging 
+
+Datadog is a tool for correlating machine data across multiple dimensions of cardinality. It cross-references an individual data points regardless of type. For example, you can compare database host telemetry with web server host thread pool sizes, displaying all logs in-line. 
```

Comment:
> Datadog is a tool for correlating machine data across multiple dimensions of cardinality. It cross-references an individual data points regardless of type. For example, you can compare database host telemetry with web server host thread pool sizes, displaying all logs in-line.

I might word this as "Datadog is a tool for correlating telemetry data from multiple sources based on related dimensions. Those dimensions are defined by tags. This allows, for example, comparison between database host metrics, with web server host thread pools, with logs from both hosts, all in a single view."